### PR TITLE
Translating conditional-rendering.md

### PR DESCRIPTION
**Pull request by AdhamMoussa**
- Translating conditional_rendering page (Done)

### CODE REVIEW

**Comment by AdhamMoussa** [comment]
@asantarissy Ready for review

File: content/docs/conditional-rendering.md [hunk 1]
```diff
@@ -116,13 +116,13 @@ ReactDOM.render(
...
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
+[**جرّب المثال على موقع CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
 
-While declaring a variable and using an `if` statement is a fine way to conditionally render a component, sometimes you might want to use a shorter syntax. There are a few ways to inline conditions in JSX, explained below.
+بينما يعتبر استخدام المتغيرات والتعبير الشرطي if هي طريقه سليمه لعمل التصيير الشرطي للمكوّن، إلا أنّك في بعض الأحيان قد ترغب في استخدام صياغه أقصر. هناك بعض الطرق تمكنك من استخدام التعبير الشرطي المباشر في JSX، شرح هذه الطرق بالأسفل.
 
-### Inline If with Logical && Operator {#inline-if-with-logical--operator}
+### التعبير الشرطي المباشر بإستخدام معامل AND المنطقي {#inline-if-with-logical--operator}
```
Comment (iRayan7) [inline content/docs/conditional-rendering.md]: AND -> &&

File: content/docs/conditional-rendering.md [hunk 2]
```diff
@@ -116,13 +116,13 @@ ReactDOM.render(
...
 
-You may [embed any expressions in JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) by wrapping them in curly braces. This includes the JavaScript logical `&&` operator. It can be handy for conditionally including an element:
+يمكنك [تضمين أيّ تعبيرات في JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) من خلال تغليفهم داخل القوسان المعقوصان `{}`. يتضمن هذا معامل AND `&&` المنطقي في JavaScript. قد يصبح هذا سهل الاستخدام لتضمين عنصر بشكل شرطي:
```
Comment (iRayan7) [inline content/docs/conditional-rendering.md]: No need to add `AND` to describe `&&`

**Comment by iRayan7** [comment]
Salam Adham, great job! 🎉
Please check out the notes I added. Thanks.

**Comment by iRayan7** [comment]
🎉

**Pull request merged**